server/app/services/calculation.service.js

```diff
@@ -38,7 +38,7 @@ const post = async item => {
     request.output("id", mssql.Int, null);
     const response = await request.execute("Calculation_Insert");
 
-    return response.outputParameters;
+    return response.output;
```

Comment:
This was returning an undefined variable so I used the VSCode debugger to check what the property was supposed to be - it's been rectified now and actually returns the `id`

Reply:
Nice!